### PR TITLE
Add Team Performance Metrics Columns with Aggregations by Gameweek

### PR DESCRIPTION
**Pull request by islameehassan**
Added the following columns: Points, Goals Scored, Goals Conceded, and Goals Difference(Scored-Conceded). Each metric is aggregated starting from the first week until the current week in the season. These metrics can add extra insights about the performance of the two teams playing. 

This new file was generated by modifying the clean_merged_seasons.csv, not each season's cleaned gw file. I decided to keep the clean_merged_seasons.csv as is and export a new csv file with my modifications, under the name "clean_merged_seasons_with_team_aggregated.csv".

[notebook](https://colab.research.google.com/drive/1uAaUb5413p5ScdRNQMZklJrmcweVaoqc)

### CODE REVIEW

**Comment by vaastav** [comment]
Hi @islameehassan , I am happy to merge this if you can also include the code for generating this file.

**Comment by islameehassan** [comment]
> Hi @islameehassan , I am happy to merge this if you can also include the code for generating this file.

Hi @vaastav . Do you want the code to be merged in the home dir or the data dir?

**Comment by vaastav** [comment]
In the home directory along with the other .py files. Thanks!

**Comment by islameehassan** [comment]
> In the home directory along with the other .py files. Thanks!

done.

File: aggregated_points_goals.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    for index,row in team_matches.iterrows():
+        team_matches.loc[index,'team_goals_scored'] = team_matches.loc[index,'team_h_score'] if row['was_home'] else team_matches.loc[index,'team_a_score']
+        team_matches.loc[index,'team_goals_conceded'] = team_matches.loc[index,'team_a_score'] if row['was_home'] else team_matches.loc[index,'team_h_score']
+
+    team_matches['team_goals_scored'] = team_matches.groupby(['team_x','season_x'])['team_goals_scored'].cumsum()
+    team_matches['team_goals_conceded'] = team_matches.groupby(['team_x','season_x'])['team_goals_conceded'].cumsum()
+    team_matches['team_goals_diff'] = team_matches['team_goals_scored'] - team_matches['team_goals_conceded']
+    newData = data.merge(team_matches[['season_x','GW','team_x','points','team_goals_scored','team_goals_conceded','team_goals_diff']], on=['season_x','GW','team_x'],how='left')
+    return newData
+
```
Comment (vaastav) [inline aggregated_points_goals.py:30]: Could you also please include a main function that reads the input data from the required csv file, executes modifyTable, and then creates the new csv file.

Comment (islameehassan) [reply aggregated_points_goals.py]: Added main and re-created the file under "cleaned_merged_seasons_team_aggregated.csv"

**Pull request merged**